install_config/topics/configuring_for_azure_registry.adoc

```diff
@@ -0,0 +1,288 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-registry_{context}']
+= Configuring the {product-title} registry for Microsoft Azure
+
+Microsoft Azure provides object cloud storage that {product-title}
+can use to store container images using the {product-title} container registry.
+
+For more information, see link:https://azure.microsoft.com/en-us/services/storage/blobs/[Cloud Storage in the Azure documentation].
+
+.Prerequisites
+
+You must create a storage account to host the registry images before
+installation. The following command creates a storage account which is used
```

Comment:
This is to create the registry but it may be worth being part of the larger before you begin because it can be applicable before performing the installation or if you are replacing the registry post installation. Most likely a customer will do it in the beginning.

Reply:
@bfallonf s/creates a storage account which is used/creates a storage account that is used or creates a storage account, which is used